app/views/shared/_category_card.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="category">
+  <div class="category_card">
+    <div class="icon_card">
+      <%= image_tag(category.icon, class: "icon") if category.icon.attached? %>
+    </div>
+    <div class= "category_info">
+      <%= category.name %>
+    </div>
+  </div>
+</div>
```

Comment:
- [ ] Currently, I'm not able to see the amount. Please show the amount as well of the transaction and in the category index show the total amount of transaction as per the design. (Both are missing the amount)

### Category index
![Screenshot from 2022-08-14 00-40-28](https://user-images.githubusercontent.com/26767383/184507566-8392c752-5e2a-42ba-bfd4-09adf951d8b4.png)

### Category show page
![Screenshot from 2022-08-14 00-30-03](https://user-images.githubusercontent.com/26767383/184507546-186e97ac-76b0-4e38-9af6-533deb47edb5.png)